WordPress/WordPressApi/WordPressComRestApi.swift

```diff
@@ -244,12 +249,19 @@ final class WordPressComRestAPIResponseSerializer: AFJSONResponseSerializer
     }
 
     override func responseObjectForResponse(response: NSURLResponse?, data: NSData?, error: NSErrorPointer) -> AnyObject? {
-        let responseObject = super.responseObjectForResponse(response, data: data, error: error)
+        var optionalError: NSError?
+        let responseObject = super.responseObjectForResponse(response, data: data, error: &optionalError)
+        error.memory = optionalError
```

Comment:
From our chat yesterday, we talked about removing the custom `acceptableStatusCodes`. In that case, we shouldn't need to manually set the error, as the `super` implementation would already do it.